### ISSUE TITLE
Missing expectation for "`video` element visual-only content has audio track alternative"

### ISSUE THREAD

**Issue by Brynanders** [act-rules/act-rules.github.io#957]
https://act-rules.github.io/rules/d7ba54 is missing information in the Expectation about the audio track being "visible and included in the accessibility" tree like in the other media rules e.g. https://act-rules.github.io/rules/2eb176

**Comment by jkodu** [comment]
@ShadowBB any progress on this?

**Comment by WilcoFiers** [comment]
I'm not sure how this is an issue. Audio tracks are not visible. I am closing this issue, since I don't see how it might be valid.
